src/lib/storage.ts

```diff
@@ -26,3 +38,47 @@ export class PreferencesStorageProvider implements StorageProvider {
 
 // singleton instance of PreferencesStorageProvider
 export const preferencesStorageProvider = new PreferencesStorageProvider()
+
+const initSqlPlugin = async () => {
+  const platform = Capacitor.getPlatform()
+
+  if (window !== undefined) {
+    jeepSqlite(window)
+
+    if (platform === "web") {
+      const jeepEl = document.createElement("jeep-sqlite")
+      document.body.appendChild(jeepEl)
+      await customElements.whenDefined("jeep-sqlite")
+      jeepEl.autoSave = true
+    }
+  }
+
+  try {
+    if (platform === "web") {
+      await sqliteService.sqliteConnection.initWebStore()
+    }
+  } catch (err: any) {
+    throw new Error(`Error while initializing SQL plugin: ${err.message || err}`)
+  }
```

Comment:
I was thinking we should just use indexeddb directly for web since sqlite only adds new layers of abstraction, but bundle size doesn't seem to be affected much, and the new storage adapter actually feels a little faster. So let's go for it.